### PR TITLE
Adjusting cosmic fcls to compressDetStepMCs and other cleanups

### PR DESCRIPTION
**Pull request by oksuzian**

### CODE REVIEW

**Comment by oksuzian** [comment]
@FNALbuild run build test

File: JobConfig/common/mu2eg4.fcl [hunk 1]
```diff
@@ -22,13 +22,168 @@ mu2e.physics.producers.g4run.commonBase: {
...
+   pars:
+   [
+	{ type: intersection pars: [
+	{ type: kineticEnergy cut: 80.0 },
+ 	{ type: pdgId pars: [ 22, 11, -11, 2112 ] }]},
+ 	{ type: union pars: [ @local::crvOuterBox ] }
+   ]
+}
+
+Cosmic:{
```
Comment (brownd1978) [inline JobConfig/common/mu2eg4.fcl:112]: Cosmic block should be in JobConfig/cosmic/prolog.fcl.   mu2eg4 should be general G4 config used by multiple parts of JobConfig.

File: JobConfig/common/mu2eg4.fcl [hunk 2]
```diff
@@ -22,13 +22,168 @@ mu2e.physics.producers.g4run.commonBase: {
    Mu2eG4CommonCut: {}
 }
 
+
```
Comment (brownd1978) [inline JobConfig/common/mu2eg4.fcl:25]: I am unsure if these parts belong in mu2eg4 or cosmic/prolog.  I believe they are only used in comsic, so cosmic/prolog is probably better.

File: JobConfig/common/mu2eg4.fcl (hunk 1 above)
Comment (oksuzian) [reply JobConfig/common/mu2eg4.fcl:112]: I thought you wanted cosmic portion defined in mu2eg4.
I'll move everything to JobConfig/cosmic/prolog.fcl, and include it from primary/prolog.fcl
That's what I originally had in mind.

**Comment by oksuzian** [comment]
@FNALbuild run build test

**Review by brownd1978 — APPROVED** [review]
Sorry for the confusion about which prolog to use, this looks good now.

**Pull request merged**